data/synapse-data-mongodb/src/main/java/io/americanexpress/synapse/data/mongodb/config/BaseReactiveMongoDBDataConfig.java

```diff
@@ -42,14 +42,18 @@ protected BaseReactiveMongoDBDataConfig(Environment environment) {
         this.environment = environment;
     }
 
+    protected String getDatabaseUri() {
```

Comment:
can you move this method to the BaseMongoDBConfig so both classes can just inherit it?